### ISSUE TITLE
Global executable

### ISSUE THREAD

**Issue by SamyPesse** [hughrawlinson/tracker-machine-battery#1]
Like this one https://github.com/Reportr/tracker-machine

Change the binary name at this line (https://github.com/hughrawlinson/tracker-machine-battery/blob/master/package.json#L16) to something like `tracker-machine-battery`.
Then you can change the installation instruction to just do `npm install -g ...`, and use directly `tracker-machine-battery`.

It could be also cool if the configuration could come from the arguments given to the executable instead of a config file in the node module. You can use something like `commander` fro this.

**Comment by SamyPesse** [comment]
BTW: nice tracker ;)

**Comment by hughrawlinson** [comment]
I did think about using commander, but I decided I would leave it out as for this to be executed by cron, you'd need to keep your password for Reportr in your crontab, which is hardly a secure way of going about things. Maybe generating authorization tokens in Reportr for specific event types would mitigate this. I'd prefer to use cron than run a node process with a setInterval call, it seems more unixy to me. I'll definitely change the installation and put it on NPM though!

Thanks
